0.6-dev/index.bs

```diff
@@ -0,0 +1,1787 @@
+Transformations between different images MUST be stored in the attributes of a parent zarr group. For transformations that store
+data or parameters in a zarr array, those zarr arrays SHOULD be stored in a zarr group `"coordinateTransformations"`.
+
+<pre>
+store.zarr                      # Root folder of the zarr store
+│
+├── zarr.json                   # coordinate transformations describing the relationship between two image coordinate systems
+│                               # are stored in the attributes of their parent group.
+│                               # transformations between 'volume' and 'crop' coordinate systems are stored here.
+│
+├── coordinateTransformations   # transformations that use array storage go in a "coordinateTransformations" zarr group.
+│   └── displacements           # for example, a zarr array containing a displacement field
+│       └── zarr.json
+│
+├── volume
+│   ├── zarr.json               # group level attributes (multiscales)
+│   └── 0                       # a group containing the 0th scale
+│       └── image               # a zarr array
+│           └── zarr.json       # physical coordinate system and transformations here
+│                               # the array attributes
+└── crop
+    ├── .zattrs                 # group level attributes (multiscales)
+    └── 0                       # a group containing the 0th scale
+        └── image               # a zarr array
+            └── zarr.json       # physical coordinate system and transformations here
+                                # the array attributes
+</pre>
+
+### Additional details
+
+Most coordinate transformations MUST specify their input and output coordinate systems using `input` and `output` with a string value
+corresponding to the name of a coordinate system. The coordinate system's name may be the path to an array, and therefore may
+not appear in the list of coordinate systems.
+
+Exceptions are if the the coordinate transformation appears in the `transformations` list of a `sequence` or is the
+`transformation` of an `inverseOf` transformation. In these two cases input and output SHOULD be omitted (see below for
+details).
+
+Transformations in the `transformations` list of a `byDimensions` transformation MUST provide `input` and `output` as arrays
+of strings corresponding to axis names of the parent transformation's input and output coordinate systems (see below for
+details).
+
+<div class=example>
+
+The sequence transformation's input corresponds to an array coordinate system at path "my/array".
+
+```json
+"coordinateSystems" : [
+    { "name" : "in", "axes" : [{"name" : "j"}, {"name":"i"}] },
+    { "name" : "outScale", "axes" : [{"name" : "y"}, {"name":"x"}] },
+    { "name" : "outSeq", "axes" : [{"name" : "y"}, {"name":"x"}] },
+    { "name" : "outInv", "axes" : [{"name" : "y"}, {"name":"x"}] },
+    { "name" : "outByDim", "axes" : [{"name" : "y"}, {"name":"x"}] }
+],
+"coordinateTransformations" : [
```

Comment:
is this an example of an excerpt of ome-zarr metadata? because this array of transformations does not compose (the input and output axes are not consistent). is that not a requirement?